src/Abstractions/NexusMods.Abstractions.Loadouts/NexusMods.Abstractions.Loadouts.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=models/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
This rule allows items in the `Models` folder to have a namespace differing from file location.
Just confirming if it's intentional.

If it isn't, can fixup in a successive PR. Not this one, as it will create havoc for visited files on my end; as I'm checking every file out of the 325 off one by one.

Reply:
Yeah, it was intentional, the idea is to not have `Models` as part of the namespace

Reply:
:ok_hand: